src/main/java/org/opencds/cqf/tooling/acceleratorkit/StructureDefinitionBaseVisitor.java

```diff
@@ -0,0 +1,40 @@
+package org.opencds.cqf.tooling.acceleratorkit;
+
+import org.hl7.fhir.r4.model.ElementDefinition;
+import org.hl7.fhir.r4.model.StructureDefinition;
+
+import java.util.List;
+
+public class StructureDefinitionBaseVisitor {
+
+    public void visitContact(StructureDefinition sd) {
```

Comment:
This class is a candidate to be abstract. Just an option to consider, no big deal!